model/fieldtypes/DBFloat.php

```diff
@@ -60,3 +60,7 @@ public function prepValueForDB($value) {
 	}
 
 }
+
+if (PHP_MAJOR_VERSION < 7) {
+	include_once 'compat/Float.php';
```

Comment:
Oh, I hadn't realised. I thought PHP7 would safely escape at the "return". Can someone confirm this?